charts/latest/csi-driver-smb/templates/csi-smb-controller.yaml

```diff
@@ -27,6 +27,10 @@ spec:
           operator: "Equal"
           value: "true"
           effect: "NoSchedule"
+        - key: "node-role.kubernetes.io/controlplane"
```

Comment:
also add this config in deploy/csi-smb-controller.yaml config

Reply:
and what k8s version is this label used?

Reply:
this was introduced in `v1.20`
https://kubernetes.io/docs/setup/release/notes/


* The label applied to control-plane nodes "node-role.kubernetes.io/master" is now deprecated and will be removed in a future release after a GA deprecation period.

* Introduce a new label "node-role.kubernetes.io/control-plane" that will be applied in parallel to "node-role.kubernetes.io/master" until the removal of the "node-role.kubernetes.io/master" label.